### ISSUE TITLE
Run OpenSeeFaceLandmarks

### ISSUE THREAD

**Issue by namhai1810** [emilianavt/OpenSeeFace#70]
Great work bro. However, why I run file model.py got this issued that  cannot mapping **kwargs
![image](https://github.com/emilianavt/OpenSeeFace/assets/117570211/79c0cb03-a61b-4cfa-9a7f-bc83b7985fe0)
![image](https://github.com/emilianavt/OpenSeeFace/assets/117570211/7423e8b3-b15d-4e11-a884-a2b854edf373)

![image](https://github.com/emilianavt/OpenSeeFace/assets/117570211/bc2fdd81-cc15-41f9-9376-a6034bc92187)

**Comment by emilianavt** [comment]
Make sure that you are on commit c450c12ae6ffb1757f62dde3c2765da3c10f6def of geffnet.

**Comment by namhai1810** [comment]
oh could you give me version of your geffnet, I use pip install geffnet(its 1.0.2)

**Comment by emilianavt** [comment]
I'd recommend cloning it from github and installing it that way.

**Comment by namhai1810** [comment]
I have solve this problem, the main reason is the the latest geffnet. Many thanks

**Comment by namhai1810** [comment]
I have observed the output of OpenSeeFaceLandmarks during training, which is of the shape (1, 198, 28, 28), with an input shape of (1, 3, 224, 224). However, our labels consist of 66 landmarks. What does the dimension 198 represent in the output? Additionally, could you provide some guidance on how to create DataLoaders for our data, given that our labels are in the form of (x, y) coordinates of landmarks?

**Comment by emilianavt** [comment]
I recommend reading through #1. Some discussion of such topics has taken place there. Regarding your question for the shapes, referring to the `landmarks` function posted there may help.

**Comment by namhai1810** [comment]
I have created a heatmap of coordinates (x, y) based on a Gaussian distribution. However, I am encountering difficulties in converting these (x, y) coordinates into X_offset_map and Y_offset_map. Could you please provide guidance on how to create these offset maps?

Furthermore, I reviewed your landmark code and noticed a function named logit_arr to reverse the effect of the sigmoid function. Does the scale factor affect the way we create the offset map? @emilianavt

**Comment by emilianavt** [comment]
For the offset maps, you can find a nice explanation of how they work [here](https://blog.tensorflow.org/2018/05/real-time-human-pose-estimation-in.html) in the "A Technical Deep Dive" section. The specific resolution numbers are different, but the technique is the same. The sigmoid function and scale factor adjust the steepness of the gradient on the offset masks.
